### PR TITLE
Dashboard - Page load time changes

### PR DESCRIPTION
**Pull request by timotei-litespeed**

### CODE REVIEW

File: tpl/dash/dashboard.tpl.php [hunk 1]
```diff
@@ -30,6 +30,8 @@
 $ucss_count = count($this->load_queue('ucss'));
 $placeholder_queue_count = count($this->load_queue('lqip'));
 $vpi_queue_count = count($this->load_queue('vpi'));
+
+$can_page_load_time = defined('LITESPEED_SERVER_TYPE') && LITESPEED_SERVER_TYPE != 'NONE' ? true : false;
```
Comment (szepeviktor) [inline tpl/dash/dashboard.tpl.php:34]: This is getting dangerous.
I promise I will never tell anyone about it!
```suggestion
$can_page_load_time = defined('LITESPEED_SERVER_TYPE') && LITESPEED_SERVER_TYPE !== 'NONE';
```

**Comment by szepeviktor** [comment]
Easy review: https://github.com/litespeedtech/lscache_wp/pull/770/files?diff=unified&w=1

File: tpl/dash/dashboard.tpl.php (hunk 1 above)
Comment (timotei-litespeed) [reply tpl/dash/dashboard.tpl.php:34]: @szepeviktor what is hgetting dangerous? didn't get that
Also made the changes.

Comment (hi-hai) [reply tpl/dash/dashboard.tpl.php]: @szepeviktor Can you explain more on this point? TIA

Comment (timotei-litespeed) [reply tpl/dash/dashboard.tpl.php]: @szepeviktor because: you can define LITESPEED_SERVER_TYPE ?

Comment (szepeviktor) [reply tpl/dash/dashboard.tpl.php]: > defined(string $constant_name): bool

https://www.php.net/manual/en/function.defined.php

**Comment by timotei-litespeed** [comment]
@szepeviktor thank you for review. Did not realized that
Closing this PR for this PR: #774 because of the many changes done on target branch

**Pull request closed without merging**